src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Lottery();
```

Comment:
According to the checklist, you should not create several instances of the same class if you can use only one instance for your purposes. The variable name `ball` implies a single ball, but it's actually an instance of `Lottery`. Consider renaming it to `lottery` to better reflect its purpose.